evap/context_processors.py

```diff
@@ -0,0 +1,10 @@
+from django.conf import settings
+
+def feedback_email(request):
+    return {'FEEDBACK_EMAIL': settings.FEEDBACK_EMAIL}
+
+def legal_notice(request):
+    return {'LEGAL_NOTICE_LINK': settings.LEGAL_NOTICE_LINK}
+
+def tracker_url(request):
+    return {'TRACKER_URL': settings.TRACKER_URL}
```

Comment:
alternatively you could add a templatetags file "settings" and include it where needed.